### PR TITLE
README.md: fix typos, clarify `tnow`'s unit

### PR DESCRIPTION
**Pull request by mustafa0x**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -971,7 +971,7 @@ plt.show()
...
 
-This second snippet addresses a potential approximation which isn’t too accurate but might be useful in some situations. The function `predictRecall` (🍏 above) in exaxct mode evaluates the log-gamma function four times and an `exp` once. One may ask, why not use the half-life returned by `modelToPercentileDecay` and Ebbinghaus’ forgetting curve, thereby approximating the current recall probability for a fact as `2 ** (-tnow / modelToPercentileDecay(model))`? While this is likely more computationally efficient (after computing the half-life up-front), it is also less precise:
+This second snippet addresses a potential approximation which isn’t too accurate but might be useful in some situations. The function `predictRecall` (🍏 above) in exact mode evaluates the log-gamma function four times and an `exp` once. One may ask, why not use the half-life returned by `modelToPercentileDecay` and Ebbinghaus’ forgetting curve, thereby approximating the current recall probability for a fact as `2 ** (-tnow / modelToPercentileDecay(model))`? While this is likely more computationally efficient (after computing the half-life up-front), it is also less precise:
```
Comment (mustafa0x) [inline README.md:974]: The change is exaxct -> exact

(mentioned since GH failed to highlight this one)

File: README.md [hunk 2]
```diff
@@ -440,7 +440,7 @@ def _subexp(x, y):
 
 
 def _meanVarToBeta(mean, var):
-  """Fit a Beta distribution to a mean and variance. 🏈"""
+  """Fit a Beta distribution to a mean and variance."""
```
Comment (mustafa0x) [inline README.md:443]: I didn't see 🏈 referenced anywhere.

**Comment by fasiha** [comment]
Units of `tnow` is actually arbitrary—the algorithm only uses the ratio of `tnow / t` (where `t` is currently the third parameter in models), so if you start with `model = defaultModel(0.5); predictRecall(model, 1.0)`, the units on 0.5 and 1 are entirely arbitrary and depend on your convention: one hour, one day, etc.

I tend to use hours for reasonable accuracy in floating point—there are 8760 hours in a year—and to encourage people to move away from the one day minimum increment in Anki, but Ebisu is agnostic to units.

Thank you for the very thorough editing! I now realize I must have lacked spellcheck when I wrote the bulk of this. If you update the pull request to omit (or clarify) the discussion of units, I'll be most happy to merge.

**Comment by mustafa0x** [comment]
Ah, ok. Yes, I guessed hours based on one of your quiz apps.

How does this look?

**Review by fasiha — APPROVED** [review]

**Comment by fasiha** [comment]
Again, many many thanks for your generosity with your time and your hard work!

**Pull request merged**